.github/workflows/lint.yml

```diff
@@ -118,7 +118,7 @@ jobs:
             - name: Check for use of %zu, which are not supported on some libcs.
               if: always()
               run: |
-                  git grep -n "%zu" -- './*' ':(exclude).github/workflows/lint.yml' && exit 1 || exit 0
+                  git grep -n "%zu" -- './*' ':(exclude).github/workflows/lint.yml' ':(exclude)examples/rvc-app/RVC_app_state_diagram.png' && exit 1 || exit 0
```

Comment:
Should we exclude all png files instead of this specific file?

Reply:
I think so

Reply:
How about we do `git grep -I -n` instead?